src/ui/theme/components/button.ts

```diff
@@ -1,21 +1,86 @@
 const ButtonStyle = {
   // style object for base or default style
   baseStyle: {
-    borderRadius: 0,
+    borderRadius: 'none',
   },
   // styles for different sizes ("sm", "md", "lg")
   sizes: {},
   // styles for different visual variants ("outline", "solid")
   variants: {
-    outline: {
-      borderRadius: 0,
-    },
+    solid: variantSolid,
   },
-  // default values for `size` and `variant`
+  // default values for `size`, `variant`, `colorScheme`
   defaultProps: {
-    size: '',
-    variant: '',
+    size: 'md',
+    variant: 'solid',
+    colorScheme: 'light',
   },
 };
 
+/* Color Schemes:
+ ** Light, Sepia, Dark
+ * States:
+ ** Normal
+ ** Selected
+ ** Disabled
+ ** Hovered
+ */
+function variantSolid(props: any) {
+  const { colorScheme, isChecked } = props;
+  const common = {
+    _focus: {
+      'z-index': 1,
+    },
+  };
+
+  const toggleButtonBg = (uncheckedColor: string) =>
+    isChecked ? 'green.500' : uncheckedColor;
+  const toggleButtonColor = (uncheckedColor: string) =>
+    isChecked ? 'white' : uncheckedColor;
+
+  switch (colorScheme) {
+    case 'sepia':
+      return {
+        ...common,
+        bg: toggleButtonBg('rgb(246, 236, 217)'),
+        color: toggleButtonColor('rgb(17, 17, 17)'),
```

Comment:
I have a PR coming after this one that includes all the NYPL colors, so we should be able to change these hard-coded strings to values from the theme like `"ui.success.primary` for the green and `ui.gray.light` for gray etc.